recipes/tedna/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: tedna
+  version: "1.2.2"
+
+build:
+  number: 0
+
+source:
+  fn: tedna_1.2.2.tar.gz
+  url: https://urgi.versailles.inra.fr/content/download/3481/29402/file/tedna_1.2.2.tar.gz
+  md5: b60b9b974e349db692eea75aad15b5e9
+
+requirements:
+  build:
+    - google-sparsehash
+    - gcc
+    - libgcc
```

Comment:
libgcc is here not needed. Also if you build for OSX please use llvm. More information are in our Guidelines. Thanks!